tests/binaries/reference-binary.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+void foo() {}
```

Comment:
this is not needed but can be used for some tests that would need to break at some func